lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php

```diff
@@ -145,6 +149,7 @@
  *      index?: bool,
  *      criteria?: array<string, string>,
  *      alsoLoadFields?: list<string>,
+ *      enumType?: class-string<BackedEnum>|null,
```

Comment:
```suggestion
 *      enumType?: class-string<BackedEnum>,
```
I guess `enumType` cannot contain `null`